core/model/modx/modrequest.class.php

```diff
@@ -526,6 +526,12 @@ public function checkPublishStatus() {
      * @return array
      */
     public function getAllActionIDs($namespace = '') {
+        // This method is deprecated, but also still in use by the core in
+        // core/model/modx/processors/system/config.js.php, which runs on each
+        // manager page, so leaving the deprecated notice commented out until
+        // that is resolved.
+        // $this->modx->deprecated('2.3.0', 'Support for modAction has been replaced with routing based on a namespace and action name.', 'modAction support');
```

Comment:
Should that be executed?

Reply:
That's literally explained in the lines above it. 

It's a deprecated method (per the docblock), that should not be used, but every single manager request currently triggers it.

Reply:
Add a @todo to the comment so it’s easier to detect.

Reply:
Sorry. I only have seen the commented out line and forgot to read the comment.

Reply:
I've removed all of the comments. The method is already tagged with `@deprecated`. When `modAction` is removed from 3.x this method should also be removed.